pages/dkp/kommander/2.2/release-notes/2.2.0/index.md

```diff
@@ -71,6 +71,17 @@ This new predictive analytics tool provides greater support productivity, speed,
 - Checking security issues
 - Verifying workloads and clusters follow best practices
 
+## Deprecations
+
+### Flag default changes
+
+For more information on using FIPS with Konvoy, see [FIPS 140-2 Compliance](../fips/).
+The default value for flag `--with-aws-bootstrap-credentials` will be changing from `true` to `false` in version v2.3.0.
+
+### Changes in behavior
+
+See [the changes in this section.](#default-update-strategy-changed-to-delete-first-for-preprovisioned-clusters).
```

Comment:
Instead of linking I think that text should just be in here, its not a new "feature" its a change from previous behavior that we want users to know about.

Reply:
Yes, let's just move the section to here.  It's in the wrong place right now.